MainProject/Packages/manifest.json

```diff
@@ -1,9 +1,10 @@
 {
   "dependencies": {
+    "com.de-panther.webxr": "file:../../com.de-panther.webxr",
```

Comment:
Please keep the package under root Packages/webxr folder instead of com.de-panther.webxr
Thanks